orga/app/components/organization-participant/table-row.gjs

```diff
@@ -111,6 +114,21 @@ export default class TableRow extends Component {
           />
         </td>
       {{/unless}}
+
+      {{#if (gt @actionsForParticipant.length 0)}}
+        <DropdownIconTrigger
```

Comment:
dans icon-trigger.js => `{{yield}}` devient `{{yield this.close}}` . Ce qui permet ensuite